label_studio/ml/api.py

```diff
@@ -20,11 +20,33 @@
 
 logger = logging.getLogger(__name__)
 
+_ml_backend_schema = openapi.Schema(
+    type=openapi.TYPE_OBJECT,
+    properties={
+        'url': openapi.Schema(type=openapi.TYPE_STRING, description='ML backend URL'),
+        'project': openapi.Schema(type=openapi.TYPE_INTEGER, description='Project ID'),
+        'is_interactive': openapi.Schema(type=openapi.TYPE_BOOLEAN, description='Is interactive'),
+        'title': openapi.Schema(type=openapi.TYPE_STRING, description='Title'),
+        'description': openapi.Schema(type=openapi.TYPE_STRING, description='Description'),
+        'auth_method': openapi.Schema(
+            type=openapi.TYPE_STRING, description='Auth method', enum=['NONE', 'BASIC_AUTH']
+        ),
+        'basic_auth_user': openapi.Schema(type=openapi.TYPE_STRING, description='Basic auth user'),
+        'basic_auth_pass': openapi.Schema(type=openapi.TYPE_STRING, description='Basic auth password'),
+        'extra_params': openapi.Schema(type=openapi.TYPE_OBJECT, description='Extra parameters'),
+        'timeout': openapi.Schema(type=openapi.TYPE_INTEGER, description='Response model timeout'),
+    },
+    required=[],
+)
+
 
 @method_decorator(
     name='post',
     decorator=swagger_auto_schema(
         tags=['Machine Learning'],
+        x_fern_sdk_group_name='ml',
```

Comment:
should this also be in the `project` group? same question as https://github.com/HumanSignal/label-studio/pull/5925/files#r1628528060 really

Reply:
we keep method names with minimal hierarchy to make it clear and avoid potential refactoring if one day we decide that ML models should also live one level above the projects